strategy/rfp-to-videographers.md

```diff
@@ -1,154 +1,114 @@
 
-Duration
+When the music softens in the last few seconds, one person who was engrossed in his/her phone looks up from his/her screen, and realizes there is not a soul around.
 
-Short, probably 2:37 mins. (determined by music).
+On the final notes of the music, we conclude with a question on the screen.
 
-Music
+#### Scope of work
 
-First 2:37 mins. of Beethoven’s Symphony 7 Movement 2 (“allegretto”)
+###Pre-production 
+1. Propose storyboard.
+2. Suggest shooting locations.
+3. Engage figurants and actors representing age groups 20 to 60.
+4. To enhance video impact, propose cinematic effects in sync with music theme.
+5. Suggest additions or revisions to the script.
 
-#### Scope of work
+###Production
 
-- Propose finalized storyboard
-- Identify and arrange shooting venue
-- Arrange figurants and actors of all age groups (i.e. from 20 to 60)
-- Propose cinematic effects to enhance video impact, in sync with proposed music theme
-- After shooting, cooperate with HTC on the producing the final product
+###Post-production
+1. Work with HTC on the producing the final product.
 
 #### Key issues
 
-- The script is simple given the proposed duration (2:37). It is on purpose as we want to impress a slow and steady progression (accentuated by the music going crescendo). Hence it will be key to enhance the appeal of the video by incorporating cinematic effects and/or suggesting additions to the script
-- Of particular important, we want the video to be as "neutral" as possible (while obviously leading the audience to realize something feels wrong about the rapid changes by technology). This means for instance that actors engrossed with their phones should be viewed under a neutral light.
+- We want to create a slow and steady progression (accentuated by the music reaching crescendo). Hence it will be important to enhance the appeal of the video by incorporating cinematic effects.
+- We want the video to have as neutral a tone as possible, while leading the audience to realize something is wrong about the rapid changes caused by technology. 
 
-#### Structure of your proposal
+#### Elements of your proposal
 
-- Introduction, confirming your understanding of our themes, concepts and importantly, objectives of our campaign and proposed video
-- Your preliminary views on key issues (see above), comments on the script, and ideas as to cinematic effects
-- Your preliminary views on the feasibility of arranging a shooting venue
-- Your preliminary views on the the availability of actors/figurants (we may need 40-50).
-- Your considerations on potential partnership (if of interest)
-- Your rough budget (please avoid quoting a wide range)
+- Introduction, confirming your understanding of our theme, concept, and the objectives of our campaign and _A Coffee Place._
```

Comment:
```suggestion
- Introduction, confirming your understanding of our theme, concept, and the objectives of our campaign and proposed video. 
```

Propose to not refer to the specific working title but a more generic "this video", so that this RFP can be used for other video projects without requiring changing such references throughout the document.. This section "Elements of your proposal" should be the same for every video.